module/Activity/view/activity/admin-category/add.phtml

```diff
@@ -0,0 +1,107 @@
+<?php
+$this->headTitle($action);
+
+$this->headScript()
+    ->prependFile($this->basePath() . '/js/activity-create.js');
+
+$form->prepare();
+$form->setAttribute('class', 'form-activity');
+?>
+<div class="row">
+    <div class="col-md-12">
+        <h2><?= $action ?></h2>
+    </div>
+    <?= $this->form()->openTag($form) ?>
+    <div class="col-md-6">
+        <div class="row">
+            <div class="col-md-12">
+                <?php
+                $checkboxDutch = $form->get('language_dutch')
+                    ->setAttribute('onchange', 'Activity.updateForm()')
+                    ->setAttribute('id', 'dutch-check')
+                    ->setAttribute('class', 'language-control');
+                ?>
+                <div class="form-group <?= $this->bootstrapElementError($checkboxDutch) ?>">
+                    <label for="<?= $checkboxDutch->getAttribute('id') ?>" class="control-label" title="<?= $this->translate('Enable Dutch translation') ?>">
+                        <?= $this->formCheckbox($checkboxDutch) ?>
+                        <span class="flag-icon flag-icon-nl"></span>
+                        <?= $this->translate('Dutch') ?>
+                    </label>
+                    <?= $this->formElementErrors($checkboxDutch) ?>
+                </div>
+                <hr>
+            </div>
+            <div class="col-md-12">
+                <?php
+                $name = $form->get('name')
+                    ->setAttribute('disabled', 'disabled')
+                    ->setAttribute('class', 'form-control form-control-dutch')
+                    ->setAttribute('id', 'name-nl');
+                $nameValue = $name->getValue();
+                $name->setValue(is_null($nameValue) ? '' : (is_string($nameValue) ? $nameValue : $nameValue->getValueNL()));
```

Comment:
Can we simplify this experssion?
Perhaps use another variable

Reply:
There are quite some similar statements for which I would like to make the same suggestion.

Reply:
I will make it a view helper to clean up the nested ternary operators.